sample_app/sampleapp_common.c

```diff
@@ -215,27 +215,43 @@ void app_print_network_details (
 
 /************************** Subslot data *************************************/
 
+/* TODO: We do not unplug modules from slots at release or abort.
+         If we reconnect with fewer modules than in last connect,
+         the reported number of modules is the old value.
+         The ART tester does not accept that all modules are pulled at
+         release or at abort.
+         Unclear if this is a problem or not */
+
 /**
  * Get subslot application information.
  * @param p_appdata        InOut: Application state.
  * @param slot_nbr         In:    Slot number.
- * @param subslot_nbr      In:    Subslot number.
+ * @param subslot_nbr      In:    Subslot number. Range 0 - 0x9FFF.
  * @return Reference to application subslot,
- *         NULL is subslot is not found/plugged.
+ *         NULL if subslot is not found/plugged.
  */
 static app_subslot_t * app_subslot_get (
    app_data_t * p_appdata,
    uint16_t slot_nbr,
    uint16_t subslot_nbr)
 {
-   uint16_t subslot;
-   for (subslot = 0; subslot < PNET_MAX_SUBSLOTS; subslot++)
+   uint16_t subslot_array_index; /* Range 0 - (PNET_MAX_SUBSLOTS - 1) */
```

Comment:
I see that the name "sub_ix" is used at least at one time in the stack. Maybe such a shorter name is to prefer? Or at least shorted to "subslot_index".

Also I think that the range is quite clear given the conditions on the for loop. I do not think the comment adds much.

Reply:
Good point. 
Renamed to `subslot_ix`
Done